### DISCUSSION TITLE
questions about keeping an eye on things

### DISCUSSION THREAD

**Discussion by n7tae** [savoirfairelinux/opendht#699] [Q&A]
I saw the section on peer discovery, but it's not helping me much.
I would like to keep an eye on the Ham DHT network I created. Is there a way to know the unhashed keys to every value that is currently published on my network?

**Comment by aberaud** [comment]
Hi, no it's not currently possible because the values are distributed over the DHT network. The only way would be to manually query each node using a custom mechanism.
